x-pack/legacy/plugins/lens/public/drag_drop/drag_drop.tsx

```diff
@@ -121,7 +122,8 @@ export function DragDrop(props: Props) {
     setState({ ...state, isActive: false });
     setDragging(undefined);
 
-    if (onDrop) {
+    if (onDrop && droppable) {
+      trackUiEvent('drop_total');
```

Comment:
This is probably fine, but if we ever want to make this drag / drop thing more general, this would be something that requires yanking out.